sky/shell/tracing_controller.cc

```diff
@@ -23,12 +28,21 @@ const char kBaseTraceEnd[] = "]}";
 
 const char kObservatoryMethodStartTracing[] = "ext.flutter_startTracing";
 const char kObservatoryMethodStopTracing[] = "ext.flutter_stopTracing";
-const char kObservatoryMethodDownloadTraceData[] = "ext.flutter_downloadTraceData";
+const char kObservatoryMethodDownloadTraceData[] =
+    "ext.flutter_downloadTraceData";
 const char kTraceFilePathParamName[] = "trace_file_path";
 const char kObservatoryResultOk[] = "{\"success\" : true}";
 const char kObservatoryResultFail[] = "{\"success\" : false}";
 const char kTraceDataNotReady[] = "{\"status\" : \"not ready\"}";
 
+static void DoNothing() {}
```

Comment:
There should already be a DoNothing in base.  https://code.google.com/p/chromium/codesearch#chromium/src/base/bind_helpers.h&q=DoNothing&sq=package:chromium&l=586